src/python/WMCore/Storage/Backends/XRDCPImpl.py

```diff
@@ -111,6 +112,9 @@ def createStageOutCommand(self, sourcePFN, targetPFN, options=None, checksums=No
                     for initFile in initFiles:
                         copyCommand += "source %s\n" % initFile
 
+        if args.disablewriterecovery and not args.cerncastor:
+            copyCommand += "env XRD_WRITERECOVERY=0 "
```

Comment:
I thought this variable would be set for the sub-shell that gets created when running this stage out script. Instead, you have to sort of pass the command to be executed as an "argument". Thanks